app/src/main/java/com/framgia/englishconversation/screen/dialog/OptionPostFragment.java

```diff
@@ -41,8 +41,13 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 
         TimelineRepository timelineRepository = new TimelineRepository(
                 new TimelineRemoteDataSource());
-        mViewModel = new OptionPostViewModel((TimelineModel) getArguments()
-                .getParcelable(Constant.PARCEL_TIMELINE));
+        mViewModel = new OptionPostViewModel(getContext(), (TimelineModel) getArguments()
+                .getParcelable(Constant.PARCEL_TIMELINE), new OnDeleteSuccessListener() {
+            @Override
+            public void onDeletePostSuccess() {
+                onDismiss(getDialog());
```

Comment:
ERROR: 'method def' child have incorrect indentation level 16, expected level should be one of the following: 24, 28.